corehq/apps/es/users.py

```diff
@@ -30,7 +30,7 @@
 class UserES(HQESQuery):
     index = 'users'
     default_filters = {
-        'not_deleted': {"term": {"base_doc": "couchuser"}},
+        'not_deleted': {"term": {"base_doc": "CouchUser"}},
```

Comment:
This works? I remember having issues with capitalization when using ES queries and for some reason it would only accept lowercase (even when things were capitalized like "CouchUser")